### PR TITLE
docs: show table of import times and package sizes in README

### PR DESCRIPTION
**Pull request by MarcoGorelli**
see https://github.com/pola-rs/polars/tree/table-for-comparisons?tab=readme-ov-file#lightweight for how it looks

EDIT: will move the asset location to polars-static

### CODE REVIEW

**Review by stinodego — COMMENTED** [review]
Looks great! Just wondering two things:

* Are the sizes/import timings up-to-date?
* Perhaps we should use polars-static repo to upload the image (just like we use for the logo): https://github.com/pola-rs/polars-static
* Maybe this would look nicer as a dark mode plot? It looks like below to me which is not very easy on the eyes:

image

File: README.md [hunk 1]
```diff
@@ -157,11 +157,9 @@ Polars is very fast. In fact, it is one of the best performing solutions availab
 
 ### Lightweight
 
-Polars is also very lightweight. It comes with zero required dependencies, and this shows in the import times:
+Polars is also very lightweight, as shown by its relatively small package size and low import times.
```
Comment (stinodego) [inline README.md:160]: May also be worth mentioning here that Polars comes with zero required dependencies!

**Comment by ritchie46** [comment]
> Maybe this would look nicer as a dark mode plot? It looks like below to me which is not very easy on the eyes:

Do you have dark-mode in the browser? I mean, it looks fine in light-mode? :sweat_smile: 

> Perhaps we should use polars-static repo to upload the image (just like we use for the logo): https://github.com/pola-rs/polars-static

Yes, plots/images should be in polars-static.

**Comment by stinodego** [comment]
We can create a light mode and a dark mode plot and show the correct one depending on the user's theme. From the astral/uv README:

```html
<p align="center">
  <picture align="center">
    <source media="(prefers-color-scheme: dark)" srcset="https://github.com/astral-sh/uv/assets/1309177/03aa9163-1c79-4a87-a31d-7a9311ed9310">
    <source media="(prefers-color-scheme: light)" srcset="https://github.com/astral-sh/uv/assets/1309177/629e59c0-9c6e-4013-9ad4-adb2bcf5080d">
    <img alt="Shows a bar chart with benchmark results." src="https://github.com/astral-sh/uv/assets/1309177/629e59c0-9c6e-4013-9ad4-adb2bcf5080d">
  </picture>
</p>
```

**Comment by MarcoGorelli** [comment]
> Yes, plots/images should be in polars-static.

cool, will pick this up again later / there

**Pull request closed without merging**